### PR TITLE
Add MessageId to websocket message, fix ws exception

### LINKED ISSUE #3251: Real-time monitoring of new media does not work in latest 10.6.0 nightly
**Describe the bug**
Moved from release 10.5.5 to nightly 10.6.0 20200604 and found that real-time monitoring of new media in all libraries is not working. When scan library task was triggered manually, the new media were promptly picked up. I made no configuration changes to my server. Reverted back to release 10.5.5 and it worked normally again.

**System (please complete the following information):**
 - OS: Synology DSM
 - Virtualization: Docker
 - Jellyfin Version: 10.6.0 nightly 20200604
 - Storage: Local using Btrfs 

**To Reproduce**
1. Use latest nightly
2. Add new media into library, with real-time monitoring enabled
3. Wait a few minutes of media to be recognised
4. New media will not be recognised until scan library task is activated

**Expected behavior**
New media is picked at real-time automatically and added into library. 

**Logs**
Not sure if logs are going to be useful here.

### PR DESCRIPTION
**Pull request by crobibero**
**Changes**
Add MessageId to all WebSocket senders.
Add JsonNonStringKeyDictionaryConverter + Factory to handle WS exceptions. Copied from api-migration branch.

**Issues**
Fixes #3251

### CODE REVIEW

**Comment by dkanada** [comment]
Do you know what change broke this?

**Comment by crobibero** [comment]
The only things I can think of are the move to System.Text.Json or a web dep changing

**Comment by mcarlton00** [comment]
Seems to fix #3190 as well, but the frontend doesn't update properly

File: MediaBrowser.Common/Json/Converters/JsonNonStringKeyDictionaryConverter.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        /// </summary>
+        /// <param name="writer">The Utf8JsonWriter.</param>
+        /// <param name="value">The dictionary value.</param>
+        /// <param name="options">The Json serializer options.</param>
+        public override void Write(Utf8JsonWriter writer, IDictionary<TKey, TValue> value, JsonSerializerOptions options)
+        {
+            var convertedDictionary = new Dictionary<string?, TValue>(value.Count);
+            foreach (var (k, v) in value)
+            {
+                convertedDictionary[k?.ToString()] = v;
```
Comment (aled) [inline MediaBrowser.Common/Json/Converters/JsonNonStringKeyDictionaryConverter.cs:73]: Null check here?
```suggestion
                if (k != null) 
                {
                  convertedDictionary[k.ToString()] = v;
                }
```

**Comment by dkanada** [comment]
@crobibero the debug build seems to be failing.

**Pull request merged**